test/e2e/Tests/Localizers/ITestLanguageLocalizer.cs

```diff
@@ -6,7 +6,8 @@ namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E;
 enum LanguageType
 {
     DotnetIsolated,
-    PowerShell
+    PowerShell,
+    Python
 }
 
 internal interface ITestLanguageLocalizer
```

Comment:
Since you have a concrete class per language implementing this interface, wouldn't it be better to add more methods here and move all the behavior differences into language-specific implementations, instead of having many switch statements scattered through code? (There is a tradeoff here, so I will not insist it _must_ be done this way, just want to make sure you considered this and made this choice intentionally.)

Reply:
I think this hearkens back to the discussion we had on the original PowerShell PR - leaving language-specific behavior that we intend to fix in the test code, and putting the language-specific behavior that we intend to leave in the SDKs into the polymorphic classes. I personally believe that each of the if-statements based on language in the test code are issues that we should _at least_ investigate, and if we decide that we won't fix them, we can move them here or somewhere similar